src/sections/Community/Handbook/mentorships.js

```diff
@@ -96,73 +74,137 @@ const MentorshipPrograms= () => {
-              <h2>Hacktoberfest</h2>
-              <img className="poster" src={Hacktoberfest} height= "300px" width= "100px"/>
-              <p>Hacktoberfest is open to everyone in the global community. Whether you’re a developer, student learning to code, event host, or a company of any size, you can help drive the growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. </p>
-              <p>Every year in October, Hacktoberfest is held. So far, Layer5 has been a participant organization with different projects and issues such as first-timers-only, help-wanted, and good-first-issues for open-source contributors. This is to help towards completing the Hacktoberfest challenge.
-                  Also, meetups are held across October to introduce you to Layer5 project repositories and help you get started on those pull requests. 
-              </p>
-              <p>Learn more about the program and how to participate, <a href="https://hacktoberfest.digitalocean.com/">Here</a></p>
-            </div>
-            <div>
-              <h2>Linux Foundation</h2>
-              <img className="poster" src={LFX}/>
-              <p>The Linux Foundation (Mentorship Programs) are designed to help developers with the necessary skills–many of whom are first-time open source contributors–experiment, learn, and contribute effectively to open source communities. 
- Layer5, as an organization, has been participating in the Linux Foundation mentorship program since 2019.
-              </p>
-              <p>The program offers mentorships, program certification, and also stipend for the mentee.</p>
-              <p>Learn more about the program and how to participate, <a href="https://lfx.linuxfoundation.org/tools/mentorship">Here</a></p>
+            </Col>
+            <Col lg={6} md={6} sm={12}>
+              <img src={GSOD} alt="explanation" />
+            </Col>
+          </Row>
+        </div>
+
+        
+        <div className="explain-2">
+          <Row>
+            <Col lg={6} md={6} sm={12}>
+              <img src={GSOC} alt="explanation" />
+            </Col>
+            <Col lg={6} md={6} sm={12}>
+              <div className="text">
+                <h2>Google Summer of Codes</h2>
+                <p>Google Summer of Codes is a global program focused on bringing more student developers into open source software development. Students work with an open-source organization on a 10-week programming project during their break from school. Every Summer, Layer5 submits a new project idea for student developers to contribute to for a particular period. Also, stipends are awarded at the end of the program to the contributors.
+                </p>
+                <Button primary id="learn" type="button" title="Learn More" url="/programs/gsoc" />
+              </div>
+            </Col>
+          </Row>
+        </div>
```

Comment:
Instead of assigning the position manually, we can do so using CSS. So, in case of future updates, it won't cause much trouble and it looks cleaner as well.